securedrop/source_templates/index.html

```diff
@@ -62,7 +62,7 @@
     Included here so the images can preload while the user is first
     reading the page. Hidden by default. -->
     <div class="bubble">
-      <p>You appear to be using the Tor Browser. You can turn the Security Slider to High in 4 easy steps!</p>
+      <p>You appear to be using the Tor Browser. In Tor's "Security Settings" pane (accessible from the toolbar's onion-button), we reccommend turning Security Slider to High—it's easy-peasy!</p>
```

Comment:
Coolio, can dig it. 

Longer-term, text-edits and ux tweeks can hopefully be rolled-out more incrementally. I'm heartened by the enthusiasm for all my recommendations, and yes, admittedly, do squeal with glee whenever I see a full page of them implemented in code—but a good goal to target, is ongoing UX updates. With time, with time. :)